SoftLayer/CLI/autoscale/create.py

```diff
@@ -0,0 +1,127 @@
+def cli(env, **args):
+    """Order/Create a scale group."""
+    scale = AutoScaleManager(env.client)
+    network = SoftLayer.NetworkManager(env.client)
+
+    datacenter = network.get_datacenter(args.get('datacenter'))
+
+    ssh_keys = []
+    for key in args.get('key'):
+        resolver = SoftLayer.SshKeyManager(env.client).resolve_ids
+        key_id = helpers.resolve_id(resolver, key, 'SshKey')
+        ssh_keys.append(key_id)
+    scale_actions = [
+        {
+            "amount": args['policy_amount'],
+            "scaleType": args['policy_relative']
+        }
+    ]
+    policy_template = {
+        'name': args['policy_name'],
+        'policies': scale_actions
+
+    }
+    policies = []
+
+    block = []
+    number_disk = 0
+    for guest_disk in args['disk']:
+        disks = {'diskImage': {'capacity': guest_disk}, 'device': number_disk}
+        block.append(disks)
+        number_disk += 1
+
+    virt_template = {
+        'localDiskFlag': False,
+        'domain': args['domain'],
+        'hostname': args['hostname'],
+        'sshKeys': ssh_keys,
+        'postInstallScriptUri': args.get('postinstall'),
+        'operatingSystemReferenceCode': args['os'],
+        'maxMemory': args.get('memory'),
+        'datacenter': {'id': datacenter[0]['id']},
+        'startCpus': args.get('cpu'),
+        'blockDevices': block,
+        'hourlyBillingFlag': True,
+        'privateNetworkOnlyFlag': False,
+        'networkComponents': [{'maxSpeed': 100}],
+        'typeId': 1,
+        'userData': [{
+            'value': args.get('userdata')
+        }],
+        'networkVlans': [],
+
+    }
+
+    order = {
+        'name': args['name'],
+        'cooldown': args['cooldown'],
+        'maximumMemberCount': args['maximum'],
+        'minimumMemberCount': args['minimum'],
+        'regionalGroupId': args['regional'],
+        'suspendedFlag': False,
+        'balancedTerminationFlag': False,
+        'virtualGuestMemberTemplate': virt_template,
+        'virtualGuestMemberCount': 0,
+        'policies': policies.append(utils.clean_dict(policy_template)),
+        'terminationPolicyId': args['termination_policy']
+    }
+
+    if not (env.skip_confirmations or formatting.confirm(
+            "This action will incur charges on your account. Continue?")):
+        raise exceptions.CLIAbort('Aborting scale group order.')
+
+    result = scale.create(order)
+
+    table = formatting.KeyValueTable(['name', 'value'])
+    table.align['name'] = 'r'
+    table.align['value'] = 'l'
+    table.add_row(['Id', result['id']])
+    table.add_row(['Created', result['createDate']])
+    table.add_row(['Name', result['name']])
+    table.add_row(['Virtual Guest Id', result['virtualGuestMembers'][0]['virtualGuest']['id']])
+    table.add_row(['Virtual Guest domain', result['virtualGuestMembers'][0]['virtualGuest']['domain']])
+    table.add_row(['Virtual Guest hostname', result['virtualGuestMembers'][0]['virtualGuest']['hostname']])
```

Comment:
Use a for cycle to display all virtual guests created, if the user use `--min 2` only the first virtual guest will be displayed and not the second